pkg/expression/builtin_compare.go

```diff
@@ -63,6 +65,7 @@ var (
 	_ builtinFunc = &builtinLeastCmpStringAsTimeSig{}
 	_ builtinFunc = &builtinIntervalIntSig{}
 	_ builtinFunc = &builtinIntervalRealSig{}
+	_ builtinFunc = &builtinLeastVectorFloat32Sig{}
```

Comment:
Should be put along with other builtinLeastXxx

Reply:
get